rigs/realm.jl

```diff
@@ -0,0 +1,76 @@
+rig_key = "realm"
+push!(RIGS, rig_key)
+#Mappings from DAQ channel to friendlier default names
+const realm_mappings = Dict("AO0"=>"axial piezo",
+                      "AO1"=>"analogout1",
+                      "AI0"=>"axial piezo monitor",
```

Comment:
It's funny now to have this called "axial" since the scan pattern is lateral. Not a big deal, but if the name is arbitrary then something like "scan mirror" might be more descriptive.

Worth checking this with @jona125, though, who is the real expert.

Reply:
Yes, you are right. But now REALM is under developing so many signals are not decided yet.
So, I just keep the OCPI-1's signal name until most of signals and its name is fixed.
Once fixed, I will change both Imagine and ImagineInterface.

Reply:
I think it's good for using OCPI-1's name for now. Because we should calibrate the angle of mirror scanning (or even the lateral scanning position) with the electronic signal. I think we should fix this later on and like @kdw503 say it have to be Imagine and ImagineInterface both tuned together.